.github/workflows/tests.yml

```
@@ -43,7 +45,7 @@ jobs:
     runs-on: ubuntu-20.04
```

```suggestion
    runs-on: ubuntu-22.04
```

Also if fixing up tests.yml, might as well use newer image